manager/controllers/baseline_update_test.go

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import (
+	"app/base/core"
+	"app/base/utils"
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUpdateBaseline(t *testing.T) {
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+	body := UpdateBaselineRequst{
+		Name:         "test_name",
+		InventoryIDs: []SystemID{"00000000-0000-0000-0000-000000000003", "00000000-0000-0000-0000-000000000004"},
+		Config:       BaselineConfig{ToTime: "2021-01-01 12:00:00-04"},
+	}
+
+	bodyJSON, err := json.Marshal(&body)
+	if err != nil {
+		panic(err)
+	}
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("POST", "/2", bytes.NewBuffer(bodyJSON))
+	core.InitRouterWithParams(BaselineUpdateHandler, 1, "POST", "/:baseline_id").ServeHTTP(w, req)
+
```

Comment:
You should verify that `baseline_id` column was updated for 0003 and 0004 systems. In the end of the test you should revert data to the previous state. We don't want to impact other tests results. We also created helper methods to reset data. Check e.g. this: https://github.com/RedHatInsights/patchman-engine/blob/master/evaluator/evaluate_advisories_test.go#L110

Reply:
... the same for update Baseline item fields. Check that it was written as expected and then reset.